src/averageAges.js

```diff
@@ -1,64 +1,51 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+function calculateAverageAge(people) {
+  return people.reduce((acc, { died, born }) =>
+    acc + (died - born), 0) / people.length;
+}
+
+function isParent(people, person) {
```

Comment:
Function names should start with a verb.
You also return a whole object from this function, so it would be better to call this function `findMother`.